DEL NextGen relnotes for 02/01/23 (#530)
harness/developer-hub: release-notes/delegate.md

For FirstGen SaaS release notes, see [Harness SaaS Release Notes (FirstGen)](/docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes). For Self-Managed Enterprise Edition, see [Self-Managed Enterprise Edition release notes](/release-notes/self-managed-enterprise-edition).

> ## February 1, 2023, version 783xx

Reviewer: remember to update the version before merging. I usually forget.

Reviewer 2: Remember to update the date here and in "date:" on line 4. The "date:" on line 4 must be updated with every new release date because it is used to update the RSS.

> ## February 1, 2023, version 783xx

Author: ```suggestion
## February 6, 2023, version 78321
```

> ### Delegate version 78101

Author: ```suggestion
### Delegate version 78306
```

Harness NextGen release 783xx includes the following changes for Harness Delegate.

:::note
The repository location of the Helm chart for the NextGen delegate is changing. (DEL-5576)

The repository is being deprecated. Updates to the chart will not be made to https://app.harness.io/storage/harness-download/delegate-helm-chart/ and will not be available from that location. To ensure retrieval of the most recent Helm chart, update your repository references to https://app.harness.io/storage/harness-download/harness-helm-charts/.
:::

### New features and enhancements

This release introduces the following new features and enhancements:

> - A REST-based operation to fetch a delegate token value was introduced. The operation requires the user permission `core_delegate_edit`. For information about other delegate token operations, see [Delegate Token Resource](https://apidocs.harness.io/tag/Delegate-Token-Resource). (DEL-5634)

Reviewer: ```suggestion
- A REST-based operation to fetch a delegate token value was introduced. The operation requires the Harness user role permission `Delegate: Create/Edit`. For information about other delegate token operations, see [Delegate Token Resource](https://apidocs.harness.io/tag/Delegate-Token-Resource). (DEL-5634)
```

Reviewer: There is no user permission named `core_delegate_edit` in Harness or its API. If you go to Account Settings > Access Control > Roles and look at a role you will see 2 Delegate perms: Delegate Configurations and Delegates.

Delegate has a Create/Edit perm. I assume this is what is needed.

You might want to confirm with Xingchi Jin as this is his ticket.
